applications/GeoMechanicsApplication/custom_elements/steady_state_Pw_element.cpp

```diff
@@ -22,7 +22,8 @@ Element::Pointer SteadyStatePwElement<TDim, TNumNodes>::Create(IndexType
                                                                NodesArrayType const& ThisNodes,
                                                                PropertiesType::Pointer pProperties) const
 {
-    return Element::Pointer(new SteadyStatePwElement(NewId, this->GetGeometry().Create(ThisNodes), pProperties));
+    return Element::Pointer(new SteadyStatePwElement(NewId, this->GetGeometry().Create(ThisNodes),
+                                                     pProperties, this->GetStressStatePolicy().Clone()));
```

Comment:
Probably the only use of the StressStatePolicy for a Pw ( or a T ) element is the integration coefficient. Richard already explained that this is here due to the inheritance from UPw. Maybe we rethink a bit in the future if we separate that in some way.